realm/src/main/java/io/realm/RealmQuery.java

```diff
@@ -932,6 +932,15 @@ private boolean containsDot(String s) {
         return this;
     }
 
+    /**
+     * Negate condition.
+     * @return The query object
+     * @throws java.lang.RuntimeException Any error
```

Comment:
It's bad to say it throws RuntimeException. It should throw something more specific. RuntimeException is an interface.